tests/entities/test_run_status.py

```diff
@@ -44,8 +44,8 @@ def test_status_mappings(self):
             RunStatus.from_string("the IMPOSSIBLE status string")
 
     def test_is_terminated(self):
-        self.assertTrue(RunStatus.is_terminated(RunStatus.FAILED))
-        self.assertTrue(RunStatus.is_terminated(RunStatus.FINISHED))
-        self.assertTrue(RunStatus.is_terminated(RunStatus.KILLED))
-        self.assertFalse(RunStatus.is_terminated(RunStatus.SCHEDULED))
-        self.assertFalse(RunStatus.is_terminated(RunStatus.RUNNING))
+        assert RunStatus.is_terminated(RunStatus.FAILED)
+        assert RunStatus.is_terminated(RunStatus.FINISHED)
+        assert RunStatus.is_terminated(RunStatus.KILLED)
+        assert RunStatus.is_terminated(RunStatus.SCHEDULED)
+        assert RunStatus.is_terminated(RunStatus.RUNNING)
```

Comment:
Are you sure this is all?

<img width="572" alt="image" src="https://user-images.githubusercontent.com/17039389/186334162-e7e801b2-89fe-488d-bd6d-5bf1446ccf5a.png">